### PR TITLE
Added New Example For Generating A World

### PR DESCRIPTION
**Pull request by oscartbeaumont**
This is a new example that shows how to generate a chunk (Using prismarine-chunk) and send it to the user. I though this might be helpful for other people. If you don't want to include it for any reason it is fine but I had just written the code and wanted to share it.

### CODE REVIEW

**Comment by Gjum** [comment]
I'd recommend sending the position after the chunk packet so the chunk is loaded when the player is spawned.

> ### Player Position And Look (clientbound)
> Updates the player's position on the server. **This packet will also close the “Downloading Terrain” screen when joining/respawning.**

**Comment by roblabla** [comment]
TYVM, we definitely want an example like that ! If you could include the changes @Gjum asked for (to make sure we don't trigger some edge case), that'd be awesome

**Comment by plexigras** [comment]
also add a `package.json` file and add `prismarine-chunk` and `vec3` as dependecys

**Comment by roblabla** [comment]
the package.json should look like https://github.com/PrismarineJS/node-minecraft-protocol/blob/master/examples/client_chat/package.json

**Comment by oscartbeaumont** [comment]
I have updated the code to add a package.json and sending the player position after sending the chunk.

File: examples/server_world/mc.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const mc = require('minecraft-protocol');
+const Chunk = require('prismarine-chunk')("1.12.1");
+const Vec3 = require('vec3');
+var server = mc.createServer({
+  'online-mode': true,
+  encryption: true,
+  host: '0.0.0.0',
+  port: 25565,
```
Comment (rom1504) [inline examples/server_world/mc.js]: Can you add version:"1.12.1" here ?

**Comment by oscartbeaumont** [comment]
Done. The Minecraft version is now in the createServer function.

**Comment by oscartbeaumont** [comment]
Thanks.

**Pull request merged**